skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/BrevredigeringService.kt

```diff
@@ -53,20 +52,17 @@ data class GeneriskRedigerbarBrevdata(
     override val saksbehandlerValg: BrevbakerBrevdata,
 ) : RedigerbarBrevdata<BrevbakerBrevdata, BrevbakerBrevdata>
 
-sealed class BrevredigeringException(message: String) : Exception(message) {
-    class KanIkkeReservereBrevredigeringException(override val message: String, val response: Api.ReservasjonResponse) :
-        BrevredigeringException(message)
-
-    class ArkivertBrevException(val brevId: Long, val journalpostId: Long) :
-        BrevredigeringException("Brev med id $brevId er allerede arkivert i journalpost $journalpostId")
-
-    class BrevIkkeKlartTilSendingException(override val message: String) : BrevredigeringException(message)
-    class BrevLaastForRedigeringException(override val message: String) : BrevredigeringException(message)
-    class HarIkkeAttestantrolleException(override val message: String) : BrevredigeringException(message)
-    class KanIkkeAttestereEgetBrevException(override val message: String) : BrevredigeringException(message)
-    class KanIkkeAttestereException(override val message: String) : BrevredigeringException(message)
-    class AlleredeAttestertException(override val message: String) : BrevredigeringException(message)
-    class BrevmalFinnesIkke(override val message: String) : BrevredigeringException(message)
+sealed class BrevredigeringException(override val message: String) : Exception() {
```

Comment:
Kva er motivasjonen bak å ikkje sende med message til superklassa her?

Reply:
Motivasjonen for endringen her var at jeg så at vi alltid setter en verdi for `message` for disse exception-klassene, og at alle bortsett fra èn av dem overstyrte `message`-feltet til not-nullable. Jeg tenkte da at da kan vi like godt flytte denne overstyringen opp til BrevredigeringException, og da burde det ikke være nødvendig å sende den med som et argument til `Exception` (siden vi overstyrer `message` feltet)